src/sentry/utils/snuba.py

```diff
@@ -298,15 +298,66 @@ def transform_aliases_and_query(**kwargs):
     conditions = kwargs['conditions'] or []
     filter_keys = kwargs['filter_keys']
 
+    def parse_selected_column_condition(column):
```

Comment:
Can you take a look at this @alex-hofsteede ?

Reply:
Oops, yeah just saw this, can review tomorrow.